analyzer/src/funTest/kotlin/integration/OssReviewToolkitIntegrationTest.kt

```diff
@@ -0,0 +1,106 @@
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/gradle/build.gradle"),
+                        File(downloadDir,
+                                "analyzer/src/funTest/assets/projects/synthetic/gradle/lib-without-repo/build.gradle"),
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/gradle/lib/build.gradle"),
+                        File(downloadDir, "build.gradle"),
+                        File(downloadDir, "downloader/build.gradle"),
+                        File(downloadDir, "graph/build.gradle"),
+                        File(downloadDir, "model/build.gradle"),
+                        File(downloadDir, "scanner/build.gradle"),
+                        File(downloadDir, "utils-test/build.gradle"),
+                        File(downloadDir, "utils/build.gradle")
+                ),
+                Maven to listOf(
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/all-managers/pom.xml"),
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/maven/app/pom.xml"),
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/maven/lib/pom.xml"),
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/maven/pom.xml")
+                ),
+                NPM to listOf(
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/all-managers/package.json"),
+                        File(downloadDir,
+                                "analyzer/src/funTest/assets/projects/synthetic/npm/multiple-lockfiles/package.json"),
+                        File(downloadDir,
+                                "analyzer/src/funTest/assets/projects/synthetic/npm/no-lockfile/package.json"),
+                        File(downloadDir,
+                                "analyzer/src/funTest/assets/projects/synthetic/npm/node-modules/package.json"),
+                        File(downloadDir,
+                                "analyzer/src/funTest/assets/projects/synthetic/npm/package-lock/package.json"),
+                        File(downloadDir,
+                                "analyzer/src/funTest/assets/projects/synthetic/npm/shrinkwrap/package.json"),
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/npm/yarn/package.json")
+                ),
+                PIP to listOf(
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/all-managers/setup.py")
+                ),
+                SBT to listOf(
+                        File(downloadDir, "analyzer/src/funTest/assets/projects/synthetic/all-managers/build.sbt")
+                )
+        )
+    }
+
+    override val definitionFilesForTest by lazy {
+        mapOf(Gradle as PackageManagerFactory<PackageManager> to listOf(File(downloadDir, "analyzer/build.gradle")))
+    }
+
```

Comment:
Nit: I thought you adapted to my style 😉

Reply:
I don't really mind, do you want me to change it?

Reply:
It's not necessary, no.

Reply:
I missed this one, done.